RecoLocalCalo/HcalRecProducers/src/HcalReconstructor.cc

```diff
@@ -0,0 +1,118 @@
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/HcalRecHit/interface/HcalRecHitCollections.h"
+#include "HeterogeneousCore/SonicTriton/interface/TritonClient.h"
+#include "HeterogeneousCore/SonicCore/interface/SonicEDProducer.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "DataFormats/HcalRecHit/interface/HBHERecHit.h"
+
+class HcalReconstructor : public SonicEDProducer<TritonClient>
```

Comment:
probably want to give this a less generic name, e.g. TritonHcalReconstructor, FacileHcalReconstructor, or something like that

Reply:
replaced with FacileHcalReconstructor